fml/status_or.h

```diff
@@ -0,0 +1,81 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Passer-by comment from updating the internal build - since this is a new file, were we supposed to update BUILD.gn to specify it?

Reply:
I'm seeing build errors related to being unable to find `status_or.h`. sponge2/34a90d3c-29af-45ca-952f-6e9733c86612

Reply:
I encountered an internal build error from being unable to find this file. I've filed an issue: https://github.com/flutter/flutter/issues/136858